lang/llvm-9.0/Portfile

```diff
@@ -0,0 +1,715 @@
+master_sites            https://releases.llvm.org/${version}
+#master_sites            https://prereleases.llvm.org/${llvm_version}.0/rc2
+use_xz                  yes
+extract.suffix          .tar.xz
+distfiles               llvm-${version}.src${extract.suffix}
+worksrcdir              llvm-${version}.src
+
+if {${distfiles} ne ""} {
+    if {${subport} eq "llvm-${llvm_version}"} {
+        if {[variant_isset polly]} {
+            distfiles-append     polly-${version}.src${extract.suffix}
+        }
+    } elseif {${subport} eq "clang-${llvm_version}"} {
+        distfiles-append     cfe-${version}.src${extract.suffix} compiler-rt-${version}.src${extract.suffix} libcxx-${version}.src${extract.suffix} clang-tools-extra-${version}.src${extract.suffix}
+    } elseif {${subport} eq "lldb-${llvm_version}"} {
+        distfiles-append     cfe-${version}.src${extract.suffix} lldb-${version}.src${extract.suffix}
+    }
+}
+
+checksums           llvm-9.0.0.src.tar.xz \
+                    rmd160  fb47d140555cd728006f07d7ad12ae9cb3f8ace6 \
+                    sha256  d6a0565cf21f22e9b4353b2eb92622e8365000a9e90a16b09b56f8157eabfe84 \
+                    size    32994768 \
+                    cfe-9.0.0.src.tar.xz \
+                    rmd160  43a9fde0f388fe6d021357bb4382bef6d445f279 \
+                    sha256  7ba81eef7c22ca5da688fdf9d88c20934d2d6b40bfe150ffd338900890aa4610 \
+                    size    13533024 \
+                    compiler-rt-9.0.0.src.tar.xz \
+                    rmd160  0b80105106d7c19f806b82508b5f4af771a95ae1 \
+                    sha256  56e4cd96dd1d8c346b07b4d6b255f976570c6f2389697347a6c3dcb9e820d10e \
+                    size    1993084 \
+                    libcxx-9.0.0.src.tar.xz \
+                    rmd160  e92b03ab660c8bb598dae2893e01b57a449b29fe \
+                    sha256  3c4162972b5d3204ba47ac384aa456855a17b5e97422723d4758251acf1ed28c \
+                    size    1814388 \
+                    clang-tools-extra-9.0.0.src.tar.xz \
+                    rmd160  284072baa8c9771de7e35cb1e9ad4ed813ff20c9 \
+                    sha256  ea1c86ce352992d7b6f6649bc622f6a2707b9f8b7153e9f9181a35c76aa3ac10 \
+                    size    2183436 \
+                    lldb-9.0.0.src.tar.xz \
+                    rmd160  ec312e0ef133e8a649a833ce7ca7338d5fdae377 \
+                    sha256  1e4c2f6a1f153f4b8afa2470d2e99dab493034c1ba8b7ffbbd7600de016d0794 \
+                    size    9846624
+
+
+patch.pre_args  -p1
+patchfiles \
+    0001-Set-the-Mach-O-CPU-Subtype-to-ppc7400-when-targeting.patch \
+    0002-Define-EXC_MASK_CRASH-and-MACH_EXCEPTION_CODES-if-th.patch \
+    0003-MacPorts-Only-Don-t-embed-the-deployment-target-in-t.patch \
+    0004-Fix-build-issues-pre-Lion-due-to-missing-a-strnlen-d.patch \
+    0005-Threading-Only-call-pthread_setname_np-on-SnowLeopar.patch \
+    0006-Only-call-setpriority-PRIO_DARWIN_THREAD-0-PRIO_DARW.patch
+
+# added by kencu for older systems support
+patchfiles-append \
```

Comment:
Jeremy, these 5000-series patches are for older systems support. The could be rolled into your rebasing repo if you prefer (better).